code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -162,6 +162,12 @@
 	else
 		affecting = get_bodypart(ran_zone(user.zone_selected))
 	var/target_area = parse_zone(check_zone(user.zone_selected)) //our intended target
+	if(affecting)
+		if(I.force && I.damtype != STAMINA && affecting.status == BODYPART_ROBOTIC) // Bodpart_robotic sparks when hit, but only when it does real damage
+			if((I.force >= 5) || I.is_sharp())
+				do_sparks(1, FALSE, loc)
+			if(prob(25))

```

Comment:
Move the oil splatters under the above check, so they don't trigger below 5 force either.

Also, I'd remove that `is_sharp()` check, since if something is sharp and deals less than 5 damage it's probably shit anyway